docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -266,10 +267,10 @@ The above will sort the buckets based on the avg rating among the promoted produ
 ==== Offset
 
 By default the bucket keys start with 0 and then continue in even spaced steps of `interval`, e.g. if the interval is 10 the first buckets
-(assuming there is data inside them) will be [0 - 9], [10-19], [20-29]. The bucket boundaries can be shifted by using the `offset` option.
+(assuming there is data inside them) will be [0, 10), [10, 20), [20, 30). The bucket boundaries can be shifted by using the `offset` option.
```

Comment:
For consistency, can you format these ranges as code (i.e. within `backticks`) please?